lib/ansible_tower_client/base_model.rb

```diff
@@ -82,10 +82,9 @@ def self.create(*args)
     def update_attributes!(attributes)
       @api.patch(url, attributes.to_json)
       attributes.each do |method_name, value|
-        if override_raw_attributes[method_name]
-          send("#{override_raw_attributes[method_name]}=", value)
-        else
-          send("#{method_name}=", value)
+        invoke_name = "#{override_raw_attributes[method_name] ? override_raw_attributes[method_name] : method_name}="
+        if respond_to?(invoke_name)
+          send(invoke_name, value)
```

Comment:
Probably need a `else` or `rescue` here that logs a debug message saying it couldn't write to a specific attribute name.